string status
string[] active_trackers
string[] cursor_positions
builtin_interfaces/Time last_update 